### ISSUE TITLE
Instructor Insights: style quotes

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-course-hugo-theme#10]
#### Designs and Mockups

Before: https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-034-artificial-intelligence-fall-2010/instructor-insights/teaching-heuristics/

After: Design TBD, but you can see that the current styling is insufficient at https://ocwnext.odl.mit.edu/courses/6-034-artificial-intelligence-fall-2010/sections/instructor-insights/teaching-heuristics/

#### Acceptance Criteria:

- [x] I don't see the necessary CSS classes coming through, so this may require a change in ocw-to-hugo as well

#### Out of Scope

- #9 Instructor Insights styling

**Comment by gumaerc** [comment]
The PR's for this were merged and after they are released, quotes should be coming through as they are in the designs.
